config/routes.rb

```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+
+
+
+
+
+
+  get 'outings' => 'outings#index'
```

Comment:
@Dpalazzari We discussed this topic here with @njgheorghita and @drod1000 and propose to leave this line for now and revisit once the nested resources are implemented later on.